packages/cli/src/utils/generate.ts

```diff
@@ -437,6 +437,40 @@ function validateAndInstallMissingDependencies(basePath: string) {
   })
 }
 
+// TODO: Read the value from an environment variable
```

Comment:
When will this be done? Shouldn't we use it a `process.env.ENABLE_REDIRECTS_MIDDLEWARE` env var already?

Reply:
But if this implementation is deployed to Vercel, the middleware will run and we'll be billed.